addons/im_livechat/static/tests/embed/chat_bubble.test.js

```diff
@@ -0,0 +1,31 @@
+import {
+    defineLivechatModels,
+    loadDefaultEmbedConfig,
+} from "@im_livechat/../tests/livechat_test_helpers";
+import { contains, setupChatHub, start, startServer } from "@mail/../tests/mail_test_helpers";
+import { describe, test } from "@odoo/hoot";
+import { Command, makeMockEnv } from "@web/../tests/web_test_helpers";
+
+describe.current.tags("desktop");
+defineLivechatModels();
+
+test("Do not show bot IM status", async () => {
+    const pyEnv = await startServer();
+    await loadDefaultEmbedConfig();
+    await makeMockEnv({ embedLivechat: true });
+    const partnerId1 = pyEnv["res.partner"].create({ name: "Mitchell", im_status: "online" });
+    pyEnv["res.users"].create({ partner_id: partnerId1 });
+    const channelId1 = pyEnv["discuss.channel"].create({
+        channel_member_ids: [Command.create({ partner_id: partnerId1 })],
+        channel_type: "chat",
+    });
+    const partnerId2 = pyEnv["res.partner"].create({ name: "Dummy", im_status: "im_partner" });
+    const channelId2 = pyEnv["discuss.channel"].create({
+        channel_member_ids: [Command.create({ partner_id: partnerId2 })],
+        channel_type: "chat",
+    });
+    setupChatHub({ folded: [channelId1, channelId2] });
+    await start({ authenticateAs: false });
+    await contains(".o-mail-ChatBubble[name='Mitchell'] .o-mail-ImStatus");
+    await contains(".o-mail-ChatBubble[name='Dummy'] .o-mail-ImStatus", { count: 0 });
```

Comment:
Would assert presence of both chat bubble, and then assert presence / non-presence of IM status.

Reason is that the test may pass, not because of the lack of IM status on "Dummy" chat bubble but simply that this chat bubble is not visible at all.